packages/playwright/src/worker/testInfo.ts

```diff
@@ -31,7 +31,8 @@ import type { StackFrame } from '@protocol/channels';
 import { testInfoError } from './util';
 
 export interface TestStepInternal {
-  complete(result: { error?: Error | unknown, attachments?: Attachment[], suggestedRebaseline?: string }): void;
+  complete(result: { error?: Error | unknown, suggestedRebaseline?: string }): void;
+  attachments: Attachment[];
```

Comment:
We need to maintain single source of truth. So these should be indexes in the array, similar to the way it is done for reporter.

Reply:
done